src/util/base58.rs

```diff
@@ -66,8 +72,14 @@ pub fn from(data: &str) -> Result<Vec<u8>, Error> {
 }
 
 /// Decodes a base58check-encoded string into a byte vector verifying the checksum.
+#[deprecated(since = "0.30.0", note = "Use base58::decode_check() instead")]
 pub fn from_check(data: &str) -> Result<Vec<u8>, Error> {
-    let mut ret: Vec<u8> = from(data)?;
+    decode_check(data)
+}
+
+/// Decodes a base58check-encoded string into a byte vector verifying the checksum.
+pub fn decode_check(data: &str) -> Result<Vec<u8>, Error> {
```

Comment:
Maybe `decode_with_check` or `decode_with_checksum`?

Reply:
The name of the encoding is "base58check". Arguably we should say `decode_base58check` but that's really long and the module is already called `base58`. But adding any extra words would be less clear IMO.